environment.js

```diff
@@ -1,10 +1,12 @@
+const U256 = require('./u256.js')
+const Block = require('ethereumjs-block')
 const constants = require('./constants.js')
 const blockChain = require('./fakeBlockChain.js')
-const U256 = require('./u256.js')
 
 module.exports = class Environment {
   constructor (data) {
     const defaults = {
+      block: new Block(),
```

Comment:
I think we need to do the split of "environment" and "call environment" before mixing these fields into one.